### ISSUE TITLE
String containing escape char (and no var) incorrectly flagged

### ISSUE THREAD

**Issue by phrawzty** [rodjek/puppet-lint#26]
Hello again,

Consider :

``` ruby
class t {
    $alpha = 'foo\n'
    $delta = "bar\n"               # line 3
}
```

Will result in :
`WARNING: double quoted string containing no variables on line 3`

I've never used the Puppet Lexer, so I'm totally in the dark as to whether it can be leveraged to deal with this or not; once I get a chance to look at it I'll submit a patch, but if you have a solution, then go for it. ;)
